modules/crawler.py

```diff
@@ -220,30 +201,15 @@ def parse_images(url, soup, response):
             response = requests.get(link)
             img = Image.open(BytesIO(response.content))
             image_hash = imagehash.phash(img)
-            tag_indicators.append(add_indicator("3-image-phash", image_hash))
-        except Exception as ex:
-            continue  # print(ex.message)
+            tag_indicators.append(Indicator("3-image-phash", image_hash))
+        except Exception as e:
+            logging.error(e, exc_info=True)
+            continue
 
     return tag_indicators
 
-
-def add_verification_tags(name, content):
-    return add_indicator("1-verification_id", name + "|" + content)
-
-
-def add_meta_social_tags(name, content):
-    return add_indicator("3-meta_social", name + "|" + content)
-
-
-def add_script_src_tags(content):
-    return add_indicator("3-script_src", content)
-
-
-def add_link_tags(href):
-    return add_indicator("3-link_href", href)
```

Comment:
simplified code means we don't need these functions anymore - they were only ever used once